website/docs/guides/schema-imports.mdx

```diff
@@ -1,41 +1,63 @@
 ---
 title: Schema Imports
-sidebar_position: 1
+sidebar_position: 1.2
 ---
 
 import ApiSchema from '@theme/ApiSchema';
+import openApi from '../../openapi/using-single-json.openapi.json';
 
-# Schema Imports
+# ApiSchema
 
-You can import model definitions from your API schema and render them in your Docusaurus Docs. You'll need to create an `.mdx` file and import the React Component. Read more [here about MDX in Docusaurus](https://docusaurus.io/docs/markdown-features/react).
+You can display model definitions from your API schema and render them in your Docusaurus Docs. You'll need to create an `.mdx` file and import the React Component. Read more [here about MDX in Docusaurus](https://docusaurus.io/docs/markdown-features/react).
 
-# Import Schema Model in Docs
+:::info File format
+You cannot import a React component inside a `.md` file.  
+Change your file extension to `.mdx` before importing the React Component.  
+Read more [here about MDX in Docusaurus](https://docusaurus.io/docs/markdown-features/react).
+:::
+
+## Import
+
+```tsx
+import ApiSchema from '@theme/ApiSchema';
+```
+
+## Props
+
+| Name    | Type         | Description                                                                                          |
+| ------- | ------------ | ---------------------------------------------------------------------------------------------------- |
+| pointer | String       | the redoc reference to display in [Redoc](https://redoc.ly/docs/resources/ref-guide/#pointer) format |
+| example | boolean      | (default: false) allow to display example                                                            |
+| id      | String       | When spec not provided, load the spec from docusaurus config. Use first spec if not defined          |
+| spec    | OpenAPI spec | A JSON content spec to use                                                                           |
+| themeId | String       | redocusaurus theme to use - default to `theme-redoc`                                                 |
```

Comment:
removing this as it will go out of sync from code and no one will maintain this. types are enough to figure out props